dashel/dashel-win32.cpp

```diff
@@ -1440,18 +1443,19 @@ namespace Dashel
 	{
 		while(step(-1));
 	}
-	
+
```

Comment:
buggy indentation

Reply:
Arguably a WS-only change, but it was buggy before

Reply:
In Dashel (and Aseba), the current style is to keep indentation on empty lines. I know that currently this style is out of fashion, but then it was common. Also, I find the out-of-fashionableness somewhat arbitrary.